src/builtin_fg.cpp

```diff
@@ -69,46 +69,46 @@ int builtin_fg(parser_t &parser, io_streams_t &streams, wchar_t **argv) {
 
         builtin_print_help(parser, streams, cmd, streams.err);
 
-        j = 0;
+        job = 0;
```

Comment:
Should these not also use "nullptr" instead of "0"?

Reply:
Yup. Just pushed a commit fixing this and cleaning up the job detection in general in `fg`.